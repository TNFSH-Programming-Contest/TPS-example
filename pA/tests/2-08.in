9139 + 4675
