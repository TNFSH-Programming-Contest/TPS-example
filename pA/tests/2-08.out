9139 + 4675 = 13814
